Code/GraphMol/ROMol.h

```diff
@@ -663,6 +668,12 @@ class RDKIT_GRAPHMOL_EXPORT ROMol : public RDProps {
   BOND_BOOKMARK_MAP d_bondBookmarks;
   RingInfo *dp_ringInfo;
   CONF_SPTR_LIST d_confs;
+  std::vector<boost::shared_ptr<SGroup>> d_sgroups;
```

Comment:
Do these really need to be shared_ptrs?  There is a bit of overhead for their use.